File: app/src/main/java/com/merino/ddfilms/model/SearchResponse.java
package com.merino.ddfilms.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import lombok.Data;

@Data
public class SearchResponse {
    private int page;
    private List<Movie> results;
    @SerializedName("total_pages")
    private int totalPages;
    @SerializedName("total_results")
    private int totalResults;
}

File: app/src/main/java/com/merino/ddfilms/model/SearchResponse.kt
package com.merino.ddfilms.model

import com.google.gson.annotations.SerializedName

data class SearchResponse(
    var page: Int = 0,
    var results: List<Movie>? = null,
    @SerializedName("total_pages") var totalPages: Int = 0,
    @SerializedName("total_results") var totalResults: Int = 0
)
